R/RLearner_regr_randomForest.R

```diff
@@ -66,6 +67,18 @@ trainLearner.regr.randomForest = function(.learner, .task, .subset, .weights = N
   } else {
     data = getTaskData(.task, .subset, target.extra = TRUE)
     m = randomForest::randomForest(x = data[["data"]], y = data[["target"]], ...)
+    ## ugly hack
```

Comment:
I really dislike the way we are using `se.method = "bootstrap"`, we are creating a bagging wrapper internally. Why do we not drop `se.method = "bootstrap"` completely as it would still be possible to use a bagging wrapper on a random forest. This should result in the exact same model, or am I overlooking something?